pom.xml

```diff
@@ -1,168 +1,167 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Line endings changed? We try to standardize on unix line endings for all files - hopefully easy to set in IntelliJ